src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -1,81 +1,88 @@
-  --#{$modal-box}__body--PaddingTop: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__body--PaddingRight: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__body--PaddingLeft: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__body--last-child--PaddingBottom: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__header--body--PaddingTop: var(--#{$pf-global}--spacer--md);
+  --#{$modal-box}__body--MinHeight: calc(var(--pf-t--global--font--size--body) * var(--pf-t--global--font--line-height--body)); // Allow for at least 1 line of content in the body
+  --#{$modal-box}__body--PaddingTop: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__body--PaddingRight: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__body--PaddingLeft: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__body--last-child--PaddingBottom: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__header--body--PaddingTop: var(--pf-t--global--spacer--md);
 
   // Close
-  --#{$modal-box}__close--Top: calc(var(--#{$pf-global}--spacer--lg));
-  --#{$modal-box}__close--Right: var(--#{$pf-global}--spacer--md);
-  --#{$modal-box}__close--sibling--MarginRight: calc(var(--#{$pf-global}--spacer--xl) + var(--#{$pf-global}--spacer--sm));
+  --#{$modal-box}__close--Top: calc(var(--pf-t--global--spacer--md));
+  --#{$modal-box}__close--Right: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__close--sibling--MarginRight: calc(var(--pf-t--global--spacer--xl) + var(--pf-t--global--spacer--sm));
 
   // Footer
-  --#{$modal-box}__footer--PaddingTop: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__footer--PaddingRight: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__footer--PaddingBottom: var(--#{$pf-global}--spacer--lg);
-  --#{$modal-box}__footer--PaddingLeft: var(--#{$pf-global}--spacer--lg);
+  --#{$modal-box}__footer--PaddingTop: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__footer--PaddingRight: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__footer--PaddingBottom: var(--pf-t--global--spacer--lg);
+  --#{$modal-box}__footer--PaddingLeft: var(--pf-t--global--spacer--lg);
 
   // Footer buttons
-  --#{$modal-box}__footer--c-button--MarginRight: var(--#{$pf-global}--spacer--md); // Button spacer is used to manipulate margin-inline-start and/or margin-right values at various breakpoints, with a single value.
+  --#{$modal-box}__footer--c-button--MarginRight: var(--pf-t--global--spacer--md); // Button spacer is used to manipulate margin-inline-start and/or margin-right values at various breakpoints, with a single value.
   --#{$modal-box}__footer--c-button--sm--MarginRight: calc(var(--#{$modal-box}__footer--c-button--MarginRight) / 2);
+}
 
+.#{$modal-box} {
   position: relative;
   z-index: var(--#{$modal-box}--ZIndex);
   display: flex;
   flex-direction: column;
   width: var(--#{$modal-box}--Width);
   max-width: var(--#{$modal-box}--MaxWidth);
   max-height: var(--#{$modal-box}--MaxHeight);
+  overflow: hidden;
```

Comment:
If you have a tall thing in the modal and no modal body (the only element that allows you to scroll overflow currently), it will just run off the screen. I would probably set this to `auto` instead. 

This may cause a double scrollbar as a tall modal with a tall body element is in a super short viewport, but we can revisit that. I don't know that it would be any worse than it is now. We could also remove the overflow scroll from the modal body and make the footer sticky to the bottom.